Components/ReusableComponent/functions.php

```diff
@@ -31,6 +32,17 @@ function getACFLayout()
                 'return_format' => 'id',
                 'instructions' => 'Add or edit <a href="/wp/wp-admin/edit.php?post_type=reusable-component">reusable components</a>.'
```

Comment:
I don't seem to be able to deselect a chosen component. I could imagine it being helpful.

Reply:
Thats an easy change from acf field options, but the field is required so how a deselect would be helpful, just to clear the input? 
I will then need to trigger that event also in admin.js to clear link

Reply:
I didn't notice the field is required. Totally makes sense this way.